app/server/routes/v1/community.js

```diff
@@ -723,4 +731,209 @@ router.delete("/tags/:title", async (req, res) => {
   }
 });
 
+/**
```

Comment:
Could this be separated into its own file? I'm only mentioning it for ease of project navigation, especially concerning future teams.

Reply:
Yes I believe so. That was my plan if we were to do any more extensive css. It might mean we put a require('swagger-ui.css') as the value for customCss